cypress/integration/focus-trap-demo.spec.js

```diff
@@ -122,7 +122,29 @@ describe('focus-trap', () => {
   });
 
   describe('demo: delay', () => {
-    // TODO
+    it('activates the focus trap when delayInitialFocus is set to true', () => {
+      cy.get('#delay').should(($div) => {
+        expect($div[0].style.opacity).to.equal('0.2');
+      });
+      cy.get('#activate-delay').type('{enter}');
+      cy.get('#delay').should('have.class', 'trap is-active');
+      cy.get('#delay').should(($div) => {
+        expect($div[0].style.opacity).to.equal('1');
+      });
+      cy.get('#close-button-delay').should('have.focus');
+    });
+
+    it('activates the focus trap when delayInitialFocus is set to false', () => {
+      cy.get('#no-delay').should(($div) => {
+        expect($div[0].style.opacity).to.equal('0.2');
+      });
+      cy.get('#activate-no-delay').type('{enter}');
+      cy.get('#no-delay').should('have.class', 'trap is-active');
+      cy.get('#no-delay').should(($div) => {
+        expect($div[0].style.opacity).to.equal('1');
+      });
+      cy.get('#close-button-no-delay').should('have.focus');
+    });
```

Comment:
I know this could be simplified. But not sure if that is something you would like.

Reply:
I'm OK with what you've done there, but I would request that the tests also verify that the trap has been successfully activated by using the [verifyCrucialFocusTrapOnClicking()](https://github.com/focus-trap/focus-trap/blob/master/cypress/integration/focus-trap-demo.spec.js#L4) helper function to verify focus is now trapped as expected.